recipes/neon2sse/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+#include <NEON_2_SSE.h>
+
+
+int main(void)
+{
+    std::cout << "neon2sse test_package ran successfully." << std::endl;
+
+    return 0;
+}
```

Comment:
This "should" actually test the header + no fancy c++. 
```suggestion
#include <NEON_2_SSE.h>

#include <stdio.h>

int main()
{
    uint64_t a = 0x0102030405060708;
    uint64_t b = 0x0102030405060708;
    uint8x8_t va = vcreate_u8(a);
    uint8x8_t vb = vcreate_u8(b);
    uint8x8_t vc = vmul_u8(va, vb);
    uint64_t c;
    vst1_u8(&c, vc);
    if (c != 0x0104091019243140) {
        fprintf(stderr, "Wrong result\n");
        return 1;
    }
    return 0;
}
```

(untested + unsure whether I need to add a `UL` suffix to the literals)